package.json

```
@@ -22,9 +22,13 @@
   "exports": {
     ".": {
       "import": "./lib/index.mjs",
-      "require": "./lib/index.js"
+      "require": "./lib/index.js",
+      "types": "./types/index.d.ts"
+    },
+    "./_non-semver-use-at-your-own-risk_/*": {
+      "default": "./lib/*",
+      "types": "./types/*.d.ts"
```

I wrote this too quickly. Tested again with this code and `.d.ts` must not be specified here

```suggestion
      "types": "./types/*"
```

---

+1 done

---

I think something went wrong, I still have the old code in the diff 😅